src/rust/ide/view/graph-editor/src/component/breadcrumbs/project_name.rs

```diff
@@ -371,8 +383,10 @@ impl View for ProjectName {
     fn default_shortcuts() -> Vec<shortcut::Shortcut> {
         use shortcut::ActionType::*;
         (&[
-            (Press,   "enter",  "commit"        ),
-            (Release, "escape", "cancel_editing"),
-        ]).iter().map(|(a,b,c)|Self::self_shortcut(*a,*b,*c)).collect()
+            (      Press,           "",              "enter",          "commit"),
+            (    Release,           "",             "escape",  "cancel_editing"),
+            (DoublePress, "is_hovered",  "left-mouse-button",   "start_editing"),
+            (DoublePress, "!is_hovered", "left-mouse-button",   "cancel_editing"),
```

Comment:
Does this discard the edit on double click in  the background? If so, why? This looks strange to me

Reply:
I've replaced this with an explicit "when a node is entered" in the graph editor.
But essentially most interactions with other UI elements should probably stop the editing here. But that might be a matter for the proper focus management.